tests/pytests/unit/modules/test_postgres.py

```diff
@@ -181,3 +215,2058 @@ def test__run_initdb_with_timeout():
             with patch.dict(configmod.__opts__, {"postgres.pass": None}):
                 postgres._run_initdb("fakename", runas="saltuser")
                 cmd_run_mock.assert_called_with(cmd_str, timeout=0, **kwargs)
+
+
+def test_run_psql():
+    postgres._run_psql('echo "hi"')
+    cmd = postgres.__salt__["cmd.run_all"]
+    assert "postgres" == cmd.call_args[1]["runas"]
```

Comment:
```suggestion
    assert cmd.call_args[1]["runas"] == "postgres"
```
What we get against what we expect.

Reply:
np, done, that is a preferred syntax and I should stick with it in future.